src/pages/New.js

```diff
@@ -71,7 +72,7 @@ export default function New() {
         })}
       </Flex>
       <ModalSection>Price</ModalSection>
-      <Price value={0} />
+      <Price value={Number(priceRating)} onClick={setPriceRating} />
```

Comment:
Instead of converting the `priceRating` to a Number, I would always save a number in the state.

Required changes:
1: `useState(0)` to set the default state to `0` instead of `"0"`
2: `<Price value={priceRating} onClick={setPriceRating} />`

Reply:
implemented in https://github.com/lmachens/around-spaces/pull/50/commits/48905892023add4981507cbd3f119c7494dfa338